src/main/java/fi/okm/jod/ohjaaja/controller/profiili/ArtikkelinKatseluController.java

```diff
@@ -40,4 +39,11 @@ UUID add(
     return service.add(
         ohjaajaId, artikkelinKatseluDto.anonyymiId(), artikkelinKatseluDto.artikkeliId());
   }
+
+  @GetMapping("/viimeksi-katsellut")
+  @Operation(summary = "Gets the last viewed article ids")
+  Page<Long> getMostRecentViewedArtikkeliIds(
```

Comment:
Check [SivuDto](https://github.com/Opetushallitus/jod-yksilo/blob/main/src/main/java/fi/okm/jod/yksilo/dto/SivuDto.java) in jod-yksilo, I think we should use similar approach here to avoid english API responses.